documentation/docs/single_server_mode.md

```diff
@@ -0,0 +1,78 @@
+
+The single server mode uses an embedded SQLite database stored on a filesystem and therefore does not require access to an external database.
+
+Although it does not provide high availability through redundancy, this configuration is suitable for production and serving millions of cryptographic objects.  The server will concurrently serve requests on as many threads as available cores to the docker container.
```

Comment:
```suggestion
Although it does not provide high availability through redundancy, this configuration is suitable for production and serving millions of cryptographic objects. The server will concurrently serves requests on as many threads as available cores to the docker container.
```

Reply:
`serves`  should be `serve` (future). The rest was fixed by the latest update